projects/netgrif-components-core/src/lib/side-menu/content-components/import-net/abstract-import-net.component.spec.ts

```diff
@@ -109,7 +109,7 @@ class MyPetriNetResource {
 class TestImportComponent extends AbstractImportNetComponent {
     constructor(@Inject(NAE_SIDE_MENU_CONTROL) protected _sideMenuControl: SideMenuControl,
                 protected _petriNetResource: PetriNetResourceService,
-                protected _uriService: UriService,
+                protected _uriService: PathService,
```

Comment:
Please rename the property to `_pathService` to match the type.

Reply:
Done